frontend/src/metabase/admin/settings/components/SettingsSetting.jsx

```diff
@@ -56,7 +56,8 @@ export default class SettingsSetting extends Component {
     }
 
     renderToggleInput(setting) {
-        const on = (setting.value == null ? setting.default : setting.value) === true;
+        const value = setting.value == null ? setting.default : setting.value,
+              on    = value === true || value === "true";
```

Comment:
This is pretty minor so don't worry about changing it, but I prefer to just duplicate the `const`/`let` for every declaration. It doesn't use any more characters (unless you don't count whitespace) and looks more symetric / makes moving code around easier.